libs/core/src/lib/slider/slider.component.ts

```diff
@@ -381,7 +371,7 @@ export class SliderComponent implements OnInit, OnChanges, OnDestroy, ControlVal
     }
 
     /** @hidden */
-    getValuenow(position: number | number[], sliderValueTarget: SliderValueTargets): string | number {
+    getValueNow(position: number | number[], sliderValueTarget: SliderValueTargets): string | number {
```

Comment:
I'm not sure we are allowed to refactor components in the scope of "unit tests with ivy disabled " issue.
I would suggest creating a separate issue for that.